manual/README.md

```diff
@@ -4,7 +4,7 @@ OCAML DOCUMENTATION
 Prerequisites
 -------------
 
-- Any prerequisites required to build OCaml from sources.
+- Any prerequisites required to build the OCaml compiler itslef from sources.
```

Comment:
itslef -> itself (in fact I would remove this word completely, "the OCaml compiler" is clear enough)